Formula/cassandra@2.1.rb

```diff
@@ -105,30 +105,10 @@ def install
     (bin/"cqlsh").write_env_script libexec/"bin/cqlsh", PYTHONPATH: pypath
   end
 
-  plist_options manual: "#{HOMEBREW_PREFIX}/opt/cassandra@2.1/bin/cassandra -f"
-
-  def plist
-    <<~EOS
-      <?xml version="1.0" encoding="UTF-8"?>
-      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-      <plist version="1.0">
-        <dict>
-          <key>KeepAlive</key>
-          <true/>
-          <key>Label</key>
-          <string>#{plist_name}</string>
-          <key>ProgramArguments</key>
-          <array>
-              <string>#{opt_bin}/cassandra</string>
-              <string>-f</string>
-          </array>
-          <key>RunAtLoad</key>
-          <true/>
-          <key>WorkingDirectory</key>
-          <string>#{var}/lib/cassandra</string>
-        </dict>
-      </plist>
-    EOS
+  service do
+    run ["#{opt_bin}/cassandra", "-f"]
+    keep_alive true
+    working_dir "#{var}/lib/cassandra"
```

Comment:
```suggestion
    run [opt_bin/"cassandra", "-f"]
    keep_alive true
    working_dir var/"lib/cassandra"
```
Let's not do unnecessary interpolation. Please do the same for the other formulae below, please.

Reply:
@carlocab I'm sorry, am I missing something? `opt_bin` is a variable, so shouldn't it be `opt_bin + "/cassandra"` instead of `opt_bin/"cassandra"`

Reply:
`opt_bin` is a `Pathname`, and we can treat `Pathname`s almost like actual paths. Try `grep`ping for `opt_bin/"` in the Homebrew/core repo to see many examples.

Reply:
TIL about `/` for strings. Now it makes sense

Reply:
Not exactly -- `/` works for `Pathname`s but not `String`s.